code/modules/urist/items/guitar.dm

```diff
@@ -0,0 +1,286 @@
+//copy pasta of the violin, don't hurt me -Nien
+
+/obj/item/device/guitar
+	name = "guitar"
+	desc = "It's made of wood and has bronze strings."
+	icon = 'icons/urist/items/tools.dmi'
+	icon_state = "guitar"
+	item_state = "guitar"
+	urist_only = 1
+	force = 10
+	attack_verb = list("played metal", "made concert", "crashed", "smashed")
```

Comment:
Urist McAssistant made concert Honky McHonkerton with the guitar sounds kind of awkward. Same with played metal. Perhaps change made concert to serenaded, and change played metal to "played metal on" or something along those lines.